### PR TITLE
[EasyTest] Update EasyTest to support PHPUnit 10

### PR DESCRIPTION
**Pull request by BOB41K1987**
- Update EasyTest to support PHPUnit 10

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |

### CODE REVIEW

**Comment by BOB41K1987** [comment]
I'm not sure about 2 things
1. Do I need to modify `composer.json` for some other packages for consistency?
2. It seems like `phpunut/phpunit` dependent should be in the main `requie` block because the package depends on one of the classes from `phpunut/phpunit`

**Comment by roman-eonx** [comment]
> I'm not sure about 2 things
> 
> 1. Do I need to modify `composer.json` for some other packages for consistency?
> 2. It seems like `phpunut/phpunit` dependent should be in the main `requie` block because the package depends on one of the classes from `phpunut/phpunit`

1. Please check CI :wink: 
2. Yes, seems so. Let's try this.

**Review by alexndlm — DISMISSED** [review]

File: packages/EasyTest/src/Bridge/PhpUnit/Extension/MailerMessageLoggerListenerPhpUnitExtension.php [hunk 1]
```diff
@@ -5,14 +5,44 @@
...
+use PHPUnit\Runner\Extension\Facade;
+use PHPUnit\Runner\Extension\ParameterCollection;
+use PHPUnit\TextUI\Configuration\Configuration;
 
-final class MailerMessageLoggerListenerPhpUnitExtension implements AfterTestHook
-{
-    public function executeAfterTest(string $test, float $time): void
+if (\class_exists(TestRunnerStartedEvent::class)) {
+    /**
+     * PHPUnit >= 10.
```
Comment (alexndlm) [inline packages/EasyTest/src/Bridge/PhpUnit/Extension/MailerMessageLoggerListenerPhpUnitExtension.php:19]: Do we need to update the readme for PHPUnit 10?

Comment (BOB41K1987) [reply packages/EasyTest/src/Bridge/PhpUnit/Extension/MailerMessageLoggerListenerPhpUnitExtension.php:19]: Nice catch, thanks. I've updated, but `usage.md`, not `readme.md`

Comment (alexndlm) [reply packages/EasyTest/src/Bridge/PhpUnit/Extension/MailerMessageLoggerListenerPhpUnitExtension.php:19]: Thanks

**Review by alexndlm — APPROVED** [review]

**Review by egor-dev — APPROVED** [review]

**Review by roman-eonx — APPROVED** [review]

**Review by natepage — APPROVED** [review]
👍😎🎉

- [X] Space Review: [20](https://eonx.jetbrains.space/p/support/reviews/20/timeline)
- [X] Valid against POC16

**Pull request merged**